starcheck/calc_ccd_temps.py

```diff
@@ -292,41 +297,31 @@ def get_week_states(tstart, tstop, bs_cmds, tlm):
     :param tlm: available pitch and aacccdpt telemetry recarray from fetch
     :returns: numpy recarray of states
     """
-    cstates = Table(get_cmd_states.fetch_states(DateTime(tstart) - 30,
-                                                tstop,
-                                                vals=['obsid',
-                                                      'pitch',
-                                                      'q1', 'q2', 'q3', 'q4']))
-    # Get the last state at least 3 days before tstart and at least one hour
-    # before the last available telemetry
-    cstate0 = cstates[(cstates['tstart'] < (DateTime(tstart) - 3).secs)
-                      & (cstates['tstart'] < (tlm[-1]['date'] - 3600))][-1]
-    # get temperature data in a range around that initial state
-    ok = ((tlm['date'] >= cstate0['tstart'] - 700) &
-          (tlm['date'] <= cstate0['tstart'] + 700))
+    # Get temperature data at the end of available telemetry
+    ok = tlm['time'] > tlm['time'][-1] - 1400
     init_aacccdpt = np.mean(tlm['aacccdpt'][ok])
+    init_tlm_time = np.mean(tlm['time'][ok])
```

Comment:
While we're cleaning things up, I note that the ACIS models still use a NPNT state as a somewhat stable reference / initial state.  Should we do that in starcheck too for consistency?